File: app/src/main/java/id/assel/caribengkel/activity/main/MainViewModel.java
package id.assel.caribengkel.activity.main;

import android.app.Application;
import android.arch.lifecycle.AndroidViewModel;

import id.assel.caribengkel.activity.main.data.WorkshopLiveData;

public class MainViewModel extends AndroidViewModel {
    public MainViewModel(Application application) {
        super(application);
    }

    WorkshopLiveData workshopLocation = new WorkshopLiveData(getApplication());
}

File: app/src/main/java/id/assel/caribengkel/MainViewModel.kt
package id.assel.caribengkel

import android.app.Application
import android.arch.lifecycle.AndroidViewModel
import android.arch.lifecycle.LiveData
import android.arch.lifecycle.MutableLiveData

class MainViewModel(application: Application): AndroidViewModel(application) {
    val test: LiveData<String> = MutableLiveData<String>()
}